src/actions/roles.js

```diff
@@ -23,30 +23,6 @@ export function getRolesByNames(rolesNames: Array<string>) {
     });
 }
 
-export function getRoleByName(roleName: string) {
```

Comment:
Can you leave the action and only remove the status flag dispatches here?

Instead of
```
return bindClientFunc({	
        clientFunc: Client4.getRoleByName,	
        onRequest: RoleTypes.ROLE_BY_NAME_REQUEST,	
        onSuccess: [RoleTypes.RECEIVED_ROLE, RoleTypes.ROLE_BY_NAME_SUCCESS],	
        onFailure: RoleTypes.ROLE_BY_NAME_FAILURE,	
        params: [	
            roleName,	
        ],	
    });
```
can you change them to 
```
return bindClientFunc({	
        clientFunc: Client4.getRoleByName,	
        params: [	
            roleName,	
        ],	
    });
```

Though we are not using them in webapp and redux this issue about just about removing the status flags so i would rather have these actions still in the codebase. I feel if we are going to remove them then it should be another ticket.